frontend/src/styles/main.scss

```
@@ -0,0 +1,52 @@
+// ═══════════════════════════════════════════════════════════════════════════
+// Wurk dashboard — SCSS entry (ITCSS-ish, SRP per partial)
+//
```

Stylelint (17.14.0)

[error] Unexpected empty comment (scss/comment-no-empty)

(scss/comment-no-empty)